python/tk_multi_workfiles/browser_form.py

```diff
@@ -669,3 +675,50 @@ def _on_step_filter_changed(self, step_list):
         :param step_list: A list of Shotgun Step dictionaries.
         """
         self.step_filter_changed.emit(get_filter_from_filter_list(step_list))
+
+    # Task status filter methods
+    def __get_project_id(self, bundle):
+        """
+        Helper method to return the project id for the current context.
+
+        :param bundle: The current app bundle.
+        """
+        if bundle.tank.pipeline_configuration.is_site_configuration():
+            return None
+        else:
+            return bundle.tank.pipeline_configuration.get_project_id()
```

Comment:
Minor: no need for a "else" statement. Embrace fail-first and avoid right-indent.